src/components/Register/Username.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
I'm sure you're aware, but we definitely don't want a registration and login version of the same component.